types/experiments.ts

```diff
@@ -143,7 +146,13 @@ interface FeatureConfig {
   value: { [key: string]: unknown };
 }
 
-interface Branch {
+interface SingleFeatureBranch {
+  /**
+   * Firefox Desktop <95
+   * Firefox Android <96
+   * Firefox iOS <39
+   */
```

Comment:
Are we deprecating `SingleFeatureBranch` (mobile SDK supports both single and multiple in the same experiment).

Reply:
I think deprecating it would come as a separate schema change process later when we're not launching experiments to any single feature clients anymore.